Chainza/antelope_memento_api
# Fix/socket inconsistency

# Short description
_**What does this PR?**_

Fixes the inconsistency of websockets that may send transactions after 3 minutes instead of immediately and sometimes even not send them at all.

# Changes
- Emitting eventLog event updates the last checked block
- EventLog event is now emitted in one cycle with updating the state
- State is now emitted using timeouts instead of intervals to prevent unexpected behaviour in the future that may have been caused if the state update took more than the update interval time (currently 500ms)

# How can it be tested
Change `.env` to connect to testnet
Start the server
Connect to it via websockets as in the example providing the arguments
Start making the transactions with the provided accounts. They should be sent to you almost immediately.
